packages/loader/container-loader/src/container.ts

```diff
@@ -223,99 +221,6 @@ export interface IContainerCreateProps {
 	readonly protocolHandlerBuilder?: ProtocolHandlerBuilder;
 }
 
-/**
- * Waits until container connects to delta storage and gets up-to-date.
- *
- * Useful when resolving URIs and hitting 404, due to container being loaded from (stale) snapshot and not being
- * up to date. Host may chose to wait in such case and retry resolving URI.
- *
- * Warning: Will wait infinitely for connection to establish if there is no connection.
- * May result in deadlock if Container.disconnect() is called and never followed by a call to Container.connect().
- *
- * @returns `true`: container is up to date, it processed all the ops that were know at the time of first connection.
- *
- * `false`: storage does not provide indication of how far the client is. Container processed all the ops known to it,
- * but it maybe still behind.
- *
- * @throws an error beginning with `"Container closed"` if the container is closed before it catches up.
- * @alpha
- */
-export async function waitContainerToCatchUp(container: IContainer) {
```

Comment:
Not sure what our guidance/plan is, but I bet it would probably be best to deprecate this and remove this later.

Reply:
@tyler-cai-microsoft What's the right follow up here? If deprecation is the right next step, can we go ahead and do that?
Either way, the current state causes us to pull too much code.